Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -1663,3 +1663,13 @@
   - type: PhysicalComposition
     materialComposition:
       Plastic: 25
+
+- type: entity
+  parent: BasePlushie
+  id: PlushieMort
+  name: Morty plushie
+  description: A plushie of the lovely Morty. It's a resilient, yet sensitive type of plush.
+  components:
+    - type: Sprite
+      sprite: Objects/Fun/toys.rsi
+      state: mortplush
```

Comment:
put it in `Resources/Prototypes/DeltaV/Entities/Objects/Fun/toys.yml` and unindent the Sprite (see [ss14 conventions](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html))